### PR TITLE
Fix build procedure

### PR DESCRIPTION
**Pull request by NexusSfan**
The build procedure is now better.

### CODE REVIEW

File: src/build.sh [hunk 1]
```diff
@@ -54,7 +54,7 @@ do
 done
 
 if [ -n "${TO_INSTALL_PACKAGES}" ]; then
-    echo "These packages,${TO_INSTALL_PACKAGES}, need to be installed."
+    echo "The packages ${TO_INSTALL_PACKAGES}, need to be installed."
```
Comment (kiavash-at-home) [inline src/build.sh]: Extra space needs to be removed

File: src/build.sh [hunk 2]
```diff
@@ -155,4 +155,5 @@ sed -r "s/(Version:).*/\1 ${PACKAGE_VERSION}/" -i ${PACKAGE_NAME}/DEBIAN/control
 sed -r "s/(Architecture:).*/\1 ${ARCH}/" -i ${PACKAGE_NAME}/DEBIAN/control
 
 # build the deb binary package
-fakeroot dpkg-deb --root-owner-group -b ${PACKAGE_NAME}/ ${PACKAGE_NAME}_${PACKAGE_VERSION}_${ARCH}.deb
+dpkg-deb --root-owner-group --build ${PACKAGE_NAME}/ ${PACKAGE_NAME}_${PACKAGE_VERSION}_${ARCH}.deb
+ln -s ${PACKAGE_NAME}_${PACKAGE_VERSION}_${ARCH}.deb ${PACKAGE_NAME}.deb
```
Comment (kiavash-at-home) [inline src/build.sh]: What is the reason for the symlink?

Comment (NexusSfan) [reply src/build.sh]: Fixed for only GitHub actions to make symlink.

File: src/build.sh (hunk 1 above)
Comment (NexusSfan) [reply src/build.sh]: Good catch!

**Pull request merged**